timesketch/api/v1/resources/analysis.py

```diff
@@ -62,6 +63,47 @@ def get(self, sketch_id, timeline_id):
         return self.to_json(analysis_history)
 
 
+class AnalyzerSessionActiveListResource(resources.ResourceMixin, Resource):
+    """Resource to get analyzer session."""
+
+    @login_required
+    def get(self, sketch_id):
+        """Handles GET request to the resource.
+
+        Returns:
+            A analyzer session in JSON (instance of flask.wrappers.Response)
+        """
+        sketch = Sketch.query.get_with_acl(sketch_id)
+
+        if not sketch:
+            abort(
+                HTTP_STATUS_CODE_NOT_FOUND, 'No sketch found with this ID.')
+
+        if not sketch.has_permission(current_user, 'read'):
+            abort(
+                HTTP_STATUS_CODE_FORBIDDEN,
+                'User does not have read access to sketch')
+
+        active_sessions = sketch.get_active_analysis_sessions()
+
+        counter = collections.Counter(PENDING=0, STARTED=0, ERROR=0, DONE=0)
+        session_ids = set()
+        for session in active_sessions:
+            session_ids.add(session.id)
+            for analysis in session.analyses:
+                counter[analysis.get_status.status] += 1
+
+        schema = {
+            'objects': [{
+                'total_tasks': sum(counter.values()),
+                'total_sessions': len(active_sessions),
+                'tasks': counter,
```

Comment:
do we want to return a counter object back? or just a dict `dict(counter)` or as a list of tuples `counter.most_common()`

Reply:
it's mainly that we now need to serialize and deserialize  a counter object, rather than the more `native` dict, and I Would think the ui could just as easily read a dict object....

Reply:
Ah, I see what you mean now. So the task counter object is being serialised to a dict in the `return jsonify(schema)` so there is not a counter object in the return.

That being said, the name `tasks` is a bit misleading, I rename it to `tasks_status`